builders/deploy/dapp.md

```diff
@@ -88,7 +88,7 @@ The [Substrate template](/builders/build/templates/overview/#baseline-network-te
 To get started, select the **Substrate** template from the left-side menu. Then take the following steps:
 
 1. In the **Project Details** section, provide your project's name, your contact email, and your Telegram
-2. Enter the token decimals and symbol for your native token and the [SS58 address format](https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json){target=\_blank}
+2. Enter the token decimals and symbol for your native token and the [SS58 address format registry](https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json){target=\_blank}
```

Comment:
```suggestion
2. Enter the token decimals and symbol for your native token and the [SS58 address format](https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json){target=\_blank}
```

They're not entering the registry, they're entering the format prefix, but this links to the registry because all formats are stored in the registry.